### PR TITLE
Add special keyboard keys, KeyDown and KeyUp methods.

### PR DESCRIPTION
**Pull request by mtrea**

### CODE REVIEW

File: go/webdriver/webdriver_test.go [hunk 1]
```diff
@@ -304,8 +304,40 @@ func TestActiveElement(t *testing.T) {
...
+	}
+	defer d.Quit(ctx)
+
+	testURL, _ := testURL("webdriver_autofocus.html")
+	if err := d.NavigateTo(ctx, testURL); err != nil {
+		t.Fatal(err)
+	}
+
+	d.KeyDown(ctx, TabKey)
+	d.KeyUp(ctx, TabKey)
```
Comment (gjyalpha) [inline go/webdriver/webdriver_test.go:327]: It seems one key fact of these newly added functions is that they could support multiple keys being keydown/keyup at the same time. Could you please improve the test to cover that case as well?

File: go/webdriver/webdriver.go [hunk 2]
```diff
@@ -70,6 +129,10 @@ type WebDriver interface {
 	Capabilities() map[string]interface{}
 	// Screenshot takes a screenshot of the current browser window.
 	Screenshot(context.Context) (image.Image, error)
+	// KeyDown performs key presses to the active element.
+	KeyDown(ctx context.Context, keys string) error
+	// KeyUp releases depressed keys.
```
Comment (gjyalpha) [inline go/webdriver/webdriver.go:134]: I think it's better to improve the comments to make it clear if the semantics of the "keys" parameter means pressing/releasing the keys at the same time or one by one.

File: go/webdriver/webdriver_test.go (hunk 1 above)
Comment (mtrea) [reply go/webdriver/webdriver_test.go:327]: Ack. I'm mirroring an internal contribution. Let's follow up with a separate PR (discussed in 1:1).

File: go/webdriver/webdriver.go (hunk 2 above)
Comment (mtrea) [reply go/webdriver/webdriver.go:134]: Ack. I'm mirroring an internal contribution. Let's follow up with a separate PR (discussed in 1:1).

**Review by joshbruning — APPROVED** [review]

**Pull request merged**